docs/Summary.md

```diff
@@ -18,7 +18,7 @@ Acuminator does not perform static analysis of projects whose names contain `Tes
 | [PX1004](diagnostics/PX1004.md) | The order of view declarations will cause the creation of two cache instances. | Message | Unavailable |
 | [PX1005](diagnostics/PX1005.md) | There is probably a typo in the view delegate name. | Warning (ISV Level 3: Informational) | Available |
 | [PX1006](diagnostics/PX1006.md) | The order of view declarations will cause the creation of one cache instance for multiple DACs | Message | Unavailable |
-| [PX1007](diagnostics/PX1007.md) | A DAC, a DAC extension, and public DAC properties should have a description in the `summary` XML tag. | Warning | Available |
+| [PX1007](diagnostics/PX1007.md) | The DAC, DAC extension, or DAC property should have a description in the "summary" XML tag or "inheritdoc" XML tag | Warning | Available |
```

Comment:
We probably should change the type to `Error` because now PX1007 will report error for mapped DAC field properties of projection DACs with incorrect annotation:
```suggestion
| [PX1007](diagnostics/PX1007.md) | The DAC, DAC extension, or DAC property should have a description in the "summary" XML tag or "inheritdoc" XML tag | Error | Available |
```